etc/init.d/zfs.in

```diff
@@ -0,0 +1,74 @@
+# ZoL userland configuration.
+
+# Run `zfs mount -a` during system start?
+ZFS_MOUNT='yes'
+
+# Run `zfs unmount -a` during system stop?
+ZFS_UNMOUNT='yes'
+
+# Run `zfs share -a` during system start?
+# nb: The shareiscsi, sharenfs, and sharesmb dataset properties.
+ZFS_SHARE='yes'
```

Comment:
Not necessarily. For example, when backing up datasets to another server. I
want to keep those share properties on the dataset (for easy recovery), but
I don't want the backup server to be sharing those shares.

Whether or not sharing should be on by default is it's own discussion. But
my main point is that this is a change in default behavior that has a big
impact (sharing data over the network). That's kinda a big deal in my book
and not a change that should be made silently.

---Alex